app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -19,14 +21,14 @@ const renderNameRoleAvatar = (name, { role }) => (
   </div>
 );
 
-const renderDropdown = () => (
+const renderActionDropdown = setShowDeleteAlert => (
   <Dropdown buttonStyle="text" icon={MenuHorizontal} strategy="fixed">
     <li>Edit</li>
-    <li>Delete</li>
+    <li onClick={setShowDeleteAlert}>Delete</li>
```

Comment:
Read: 
- https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663761829787419?thread_ts=1663060378.971029&cid=C01T5Q4DZAR
- https://academy.bigbinary.com/learn-react/pages/semantic-html-and-accessibility